llvm/lib/IR/Verifier.cpp

```diff
@@ -4805,6 +4803,8 @@ void Verifier::visitInstruction(Instruction &I) {
   BasicBlock *BB = I.getParent();
   Check(BB, "Instruction not embedded in basic block!", &I);
 
+  CV.visit(I);
```

Comment:
CV should probably be renamed, I had no idea what this was

Reply:
Fixed that.